ADR/0011-roles-and-permissions.md

```diff
@@ -0,0 +1,81 @@
+# 11. Roles and Permissions for Stonesoup
```

Comment:
This conflicts with https://github.com/redhat-appstudio/book/pull/41

Reply:
the ADR number. Both are 11.

Reply:
It is a race now :)